datanode/networkhistory/service_test.go

```diff
@@ -1264,17 +1264,23 @@ func getDatabaseDataSummary(ctx context.Context, connConfig sqlstore.ConnectionC
 	}
 
 	for table, meta := range dbMetaData.TableNameToMetaData {
+		if len(meta.SortOrder) == 0 {
+			meta.SortOrder = "vega_time, seq_num"
```

Comment:
why is this needed? It should never happen, and defaulting the sort order to vega_time,seq_num is not the right thing to do as some tables have different sort order